guides/introduction/installation.md

```diff
@@ -10,4 +10,34 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, susci
 
 ## Windows
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem. Etiam pellentesque aliquet tellus. Phasellus pharetra nulla ac diam. Quisque semper justo at risus. Donec venenatis, turpis vel hendrerit interdum, dui ligula ultricies purus, sed posuere libero dui id orci. Nam congue, pede vitae dapibus aliquet, elit magna vulputate arcu, vel tempus metus leo non est. Etiam sit amet lectus quis est congue mollis. Phasellus congue lacus eget neque. Phasellus ornare, ante vitae consectetuer consequat, purus sapien ultricies dolor, et mollis pede metus eget nisi. Praesent sodales velit quis augue. Cras suscipit, urna at aliquam rhoncus, urna quam viverra nisi, in interdum massa nibh nec erat.
+This guide provide instructions for Windows 10.
```

Comment:
Can we assume also that this is compatible with Windows 8?

Reply:
This guide provides instructions for Windows 10.

Reply:
@Skoda091 I think yes, because most of Windows installations based on packages which are available for different Windows distributions. I don't have Windows 8 to validate it though.